.gitignore

```
@@ -30,3 +16,25 @@ amplifytools.xcconfig
 frontend/app/.idea/
 
 /gource.log
+
+#amplify-do-not-edit-begin
```

```suggestion
#amplify-do-not-edit-begin
amplify/
```
ik weet dat het do-not-edit zegt maar hij hoort heel /amplify mee te nemen